ui/src/components/Header.css

```diff
@@ -1,41 +1,28 @@
-.header {
-    width: 45%;
-    margin-top: 15px;
-    vertical-align: center;
-}
-
-.headerBox {
-    width: 100%;
+.toolbar {
```

Comment:
Can this be a [FAB (floating action button)](https://material.io/guidelines/components/buttons-floating-action-button.html)? You don't have to get the floating stuff working, obviously, but I would like a red button on lower right, [similar to Inbox](https://lh3.googleusercontent.com/bCf1aJaKy9CG4W8viDPPEKfwW0aE_twWRR7acRL21fJFlBsNLKT2rAKO1uxxRC5daQ=h310).

Reply:
Obsolete